packages/babel-plugin/README.md

```diff
@@ -0,0 +1,15 @@
+# @reatom/babel-plugin
+
+## Install
+
+NPM:
+
+```sh
+npm install --save-dev @reatom/babel-plugin
+```
+
+YARN:
+
+```sh
+yarn add @reatom/babel-plugin --dev
+```
```

Comment:
Too large description, it can be shorted
```suggestion
```sh
 npm install --save-dev @reatom/babel-plugin
 # OR
yarn add @reatom/babel-plugin --dev
```